src/Criteo.OpenApi.Comparator/ComparisonMessage.cs

```diff
@@ -126,48 +125,10 @@ private static string Location(IJsonDocument jsonDocument, IJsonLineInfo jsonTok
         /// </summary>
         public string NewLocation() => Location(NewDocument, NewJson());
 
-        /// <summary>
-        /// Converts JSON object into a serialized JSON string
-        /// </summary>
-        /// <returns>JSON as string</returns>
-        public string GetValidationMessagesAsJson()
-        {
-            var rawMessage = new JsonComparisonMessage
-            {
-                Id = Id.ToString(),
-                Code = Code,
-                Message = Message,
-                Type = Severity.ToString(),
-                DocUrl = DocUrl,
-                Mode = Mode.ToString(),
-                Old = new JsonLocation { Ref = OldJsonRef, Path = OldJson()?.Path, Location = OldLocation(), },
-                New = new JsonLocation { Ref = NewJsonRef, Path = NewJson()?.Path, Location = NewLocation(), }
-            };
-
-            return JsonConvert.SerializeObject(
-                rawMessage,
-                Formatting.Indented,
-                new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore }
-            );
-        }
-
         /// <summary>
         /// Converts the currents ComparisonMessage object into a string message
         /// </summary>
         public override string ToString() =>
             $"code = {Code}, type = {Severity}, message = {Message}, docurl = {DocUrl}, mode = {Mode}";
     }
-
-    public class CustomComparer : IEqualityComparer<ComparisonMessage>
```

Comment:
I don't understand the link between the MaxDepth property change explained in the commit message and changing the way the ComparisonMessage would compare.

Could you explain in the commit message and if it's an unrelated clean-up move it to another commit ?

Reply:
I guess I missed this file while splitting my commits, I'll update the message